src/hermes/cli.py

```diff
@@ -84,13 +84,16 @@ def invoke(self, ctx: click.Context) -> t.Any:
         :param ctx: Context for the command.
         """
 
-        configure(ctx.params.get('config').absolute())
+        # Get the user provided working dir from the --path option or default to current working directory.
+        working_path = ctx.params.get('path', pathlib.Path.cwd()).absolute()
```

Comment:
The path option has a default, which should be used here (`./`, i.e. the same)
```suggestion
        working_path = ctx.params.get('path').absolute()
```